libs/designer-ui/src/lib/templates/templateparameters/templateparametersField.tsx

```diff
@@ -0,0 +1,146 @@
+import type { ILabelStyles, IStyle, ITextFieldStyles } from '@fluentui/react';
```

Comment:
Is there a reason why we cant use our existing workflow parameters given that the UI is basically the same

Reply:
I tried re-using that at first, but realized I will need to add unneeded things to existing workflow parameters field if I do. Examples include:
- a description field rendering
- WorkflowParameterDefinition doesn't have description, but we'll have to add it for types purposes
- only the value field should be editable and required, meaning I have to change isEditable and isReadOnly from boolean to boolean | Record<string, boolean>
- WorkflowParameterUpdateHandler uses WorkflowParameterUpdateEvent which doesn't include 'required' field, so we will have to add it to this interface

and other formatting issues that seemed unncessary if calling WorkflowParmaretersField from Templates side.

I thought it would not make sense to add all those fields to WorkflowParametersField because they don't exist in WorkflowParameters

Reply:
What would it mean by keeping consistent with validation etc from template to designer? Validation happens at the state level, and so I had to seperate it regardless.. Also I couldn't take the whole part to the validation function since Templates only need to validate the value field. But open to suggestions!

Reply:
After offline discussion, decided to modify workflowParamaetersField and re-use this component